### ISSUE TITLE
Accessibility | Data grid Component text cell icon add accessibilityTitle

### ISSUE THREAD

**Issue by K0NZ1** [telekom/scale#2194]
**Describe the feature request**
When using the data grid component and use the icon prefix/suffix option in the text cell you can set an accessibilityTitle Attribute at the icon html tag or the accessibilityTitle will be set automatically.

**Describe the use case**
When using the data grid component and use the icon prefix/suffix option in the text cell you cannot set an accessibilityTitle Attribute at the icon html tag. Therefor the icon is not accessible for blind users when using a screen reader.

**Describe alternatives you've considered**
I discovered currently a workaround where i add the attribute afterwards.
`
const dataGrid = document.querySelector('#dataGrid')
if (dataGrid) {
  const shadowDataGrid = dataGrid.shadowRoot
  const icons = Array.from(shadowDataGrid.querySelectorAll('scale-icon-communication-available'))
  icons.forEach((icon) => {
    icon.setAttribute('accessibilityTitle', 'verfügbar')
  })
}
`
